src/query/service/src/interpreters/interpreter_delete.rs

```diff
@@ -96,17 +94,6 @@ impl Interpreter for DeleteInterpreter {
         let catalog_name = self.plan.catalog_name.as_str();
         let db_name = self.plan.database_name.as_str();
         let tbl_name = self.plan.table_name.as_str();
-
-        let tbl = self.ctx.get_table(catalog_name, db_name, tbl_name).await?;
-        let table_info = tbl.get_table_info().clone();
-
-        // Add table lock heartbeat.
-        let handler = TableLockHandlerWrapper::instance(self.ctx.clone());
```

Comment:
This pr can reduce some commit conflicts. However,  it cannot avoid the inevitable conflicts. I think add table lock is still necessary. Unless we support segment or block level locks.